aliasing/api/character.py

```diff
@@ -163,6 +163,79 @@ def create_cc(self, name: str, *args, **kwargs):
             self.delete_cc(name)
         return self.create_cc_nx(name, *args, **kwargs)
 
+    def edit_cc(self, name: str, minVal: str = None, maxVal: str = None, reset: str = None,
+                dispType: str = None, reset_to: str = None, reset_by: str = None,
+                title: str = None, desc: str = None):
+        """
+        Edits an existing custom counter.
+
+        Pass `none` to remove an argument entirely.
+        Will clamp counter value to new limits if needed.
+
+        :param str name: The name of the counter to edit.
+        :param str minVal: The minimum value of the counter. Supports :ref:`cvar-table` parsing.
+        :param str maxVal: The maximum value of the counter. Supports :ref:`cvar-table` parsing.
+        :param str reset: One of ``'short'``, ``'long'``, ``'hp'``, ``'none'``, or ``None``.
+        :param str dispType: Either ``None`` or ``'bubble'``.
+        :param str reset_to: The value the counter should reset to. Supports :ref:`cvar-table` parsing.
+        :param str reset_by: How much the counter should change by on a reset. Supports dice but not cvars.
+        :param str title: The title of the counter.
+        :param str desc: The description of the counter.
+        :rtype: AliasCustomCounter
+        :returns: The edited counter.
+        """
+        counter = self._get_consumable(name)
+
+        if minVal is not None:
+            minVal = str(minVal) if str(minVal) not in ('none', 'None') else None
+        else:
+            minVal = counter.get_min()
```

Comment:
```suggestion
        if minVal is not _UNSET:
            minVal = str(minVal) if minVal is not None else None
        else:
            minVal = counter.min
```
To support using `None` to remove a value, set the default kwarg to a special `_UNSET` sentinel value (you can define this somewhere at the top of the file as `_UNSET = object()`). It might be worth pulling this logic out into a helper function (e.g. `optional_arg_or_default(argument: Optional[T], arg_t: Type[T] = str, default: Any = None, unset_sentinel: Any = _UNSET)`).

You probably also want to use `counter.min` instead of `counter.get_min()`; the latter returns the computed value, while the former returns the expression (which is probably what we want if we want to leave it unchanged).